src/widgets/ot_pulltorefresh.eliom

```diff
@@ -70,10 +72,12 @@ module Make (Conf : CONF) = struct
       then (
         let target = ev##.changedTouches##item 0 in
         Js.Optdef.iter target (fun target ->
-            distance := !dragStart - target##.clientY);
+            distance :=
+              Float.sqrt (float_of_int (- !dragStart + target##.clientY))
```

Comment:
As you are using module Float, OT must now depend on ocaml >= 4.07.0 I think? Update opam file.

Reply:
makes sense. I didn't think about that